
This patch modifies look&feel of:

 -in-progress recording in schedule window


diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml	2022-04-26 01:09:17.000000000 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml	2022-08-14 11:35:46.387994004 +0200
@@ -469,42 +469,41 @@
             <area>25,505,900,505</area>
             <statetype name="buttonitem">
                 <state name="active">
+                    <progressbar name="buttonprogress2">
+                        <area>0,0,100%,100%</area>
+                        <layout>horizontal</layout>
+                        <style>reveal</style>
+                        <shape name="progressimage">
+                            <area>50,7,100%,90%</area>
+                            <type>box</type>
+                            <fill color="#E0991A" alpha="64"/>
+                        </shape>
+                    </progressbar>
                     <textarea name="card" from="baseitemtext">
                         <area>5,0,50,100%</area>
                         <font state="error">basefont_l_bad</font>
-                        <font state="running">basefont_l_good</font>
+                        <font state="running">basefont_l_running</font>
                         <font state="warning">basefont_l_warn</font>
                         <font state="disabled">basefont_l_off</font>
-                        <alpha>100</alpha>
+                        <alpha>100</alpha>
                     </textarea>
                     <textarea name="title" from="baseitemtext">
                         <area>50,0,60%,100%</area>
+                        <template>%title%% (|progresspercent|%)%</template>
+                        <font state="running">basefont_l_running</font>
                         <font state="disabled">basefont_l_off</font>
-                        <alpha>100</alpha>
+                        <alpha>100</alpha>
+
                     </textarea>
                     <textarea name="timedate" from="basetext_m">
                         <area>60%,0,-5,100%</area>
                         <align>right,vcenter</align>
-                        <alpha>100</alpha>
+                        <alpha>100</alpha>
                         <font state="disabled">basefont_m_off</font>
                     </textarea>
-                    <progressbar name="buttonprogress2" from="baseprogress_xs">
-                        <shape name="progressimage">
-                            <fill color="#004b00" alpha="255"/>
-                        </shape>
-                    </progressbar>
                 </state>
                 <state name="selectedactive" from="active">
-                    <area>0,0,100%,58</area>
-                    <textarea name="card">
-                        <alpha>255</alpha>
-                    </textarea>
-                    <textarea name="title">
-                        <alpha>255</alpha>
-                    </textarea>
-                    <textarea name="timedate">
-                        <alpha>255</alpha>
-                    </textarea>
+                    <alpha>255</alpha>
                 </state>
             </statetype>
         </buttonlist>
@@ -527,7 +526,7 @@
                 <area>0,100%-120,100%,120</area>
                 <!-- left side -->
                 <group name="iconfacts" from="basescheduleiconfacts">
-                    <area>0,0,50%,170</area>
+                    <area>0,0,50%,120</area>
                 </group>
                 <!-- right side -->
                 <imagetype name="group" from="baseicon" depends="recgroup">
